server.go

```diff
@@ -354,8 +356,23 @@ func (s *Server) handleQuery(query *dns.Msg, ifIndex int, from net.Addr) error {
 				err = e
 			}
 		} else {
-			// Send mulicast
-			if e := s.multicastResponse(&resp, ifIndex); e != nil {
+			// According to RFC6762 Section 6:
+			// Multicast DNS responder MUST NOT
+			// (except in the one special case of answering probe queries) multicast
+			// a record on a given interface until at least one second has elapsed
+			// since the last time that record was multicast on that particular
+			// interface
+			now := time.Now()
+			lastTime, exists := s.lastMulticast[ifIndex]
```

Comment:
I think technically we should also key the interval on the response packet too. I realize it's not an issue for us because we only have a single record per node now, but according to the RFC it isn't that we shouldn't respond more than once per second per interface, but that a given packet should not be sent more often than that:

"multicast **a record** on a given interface until at least one second has elapsed since the last time **that record** was multicast on that particular interface."